### ISSUE TITLE
NPE when loading a network

### ISSUE THREAD

**Issue by fhermeni** [btrplace/scheduler#93]
Loading the following JSON leads to a NPE
```
{"model":{"mapping":{"readyVMs":[],"onlineNodes":{"0":{"sleepingVMs":[],"runningVMs":[0]},"1":{"sleepingVMs":[],"runningVMs":[1]},"2":{"sleepingVMs":[],"runningVMs":[2]}},"offlineNodes":[]},"attributes":{"nodes":{},"vms":{"0":{"memUsed":204},"1":{"memUsed":204},"2":{"memUsed":204}}},"views":[{"defConsumption":0,"nodes":{"0":8,"1":10,"2":10},"rcId":"cpu","id":"shareableResource","defCapacity":0,"vms":{"0":8,"1":1,"2":1}},{"routing":{"type":"default"},"switches":[{"id":0,"capacity":-1}],"links":[{"physicalElement":{"id":2,"type":"node"},"id":0,"capacity":1000,"switch":0},{"physicalElement":{"id":1,"type":"node"},"id":1,"capacity":1000,"switch":0},{"physicalElement":{"id":0,"type":"node"},"id":2,"capacity":1000,"switch":0}],"id":"net"}]},"constraints":[{"rc":"cpu","amount":6,"nodes":[0],"continuous":false,"id":"resourceCapacity"}],"objective":{"id":"minimizeMTTR"}}
```

**Comment by vincent-k** [comment]
Fixed in #92

**Comment by fhermeni** [comment]
There is other issues. 
- You should add a Root constraint for VM staying running. I modified CRoot impl to force the instantiation.
- the 2nd solving pass does not work (No solution)
